packages/node_modules/@ciscospark/plugin-phone/src/call.js

```diff
@@ -949,6 +949,9 @@ const Call = SparkPlugin.extend({
   },
 
   _updateMedia(payload) {
+    if (payload.sdp && !payload.sdp.includes(`b=`)) {
```

Comment:
though, now that you mention it, that's not quite right. only video should have a b line. I can probably take this check out. it's more of a development assertion than something we actually want at runtime.